data-es6.js

```diff
@@ -8768,15 +8768,16 @@ exports.tests = [
         var get = [];
         var p = new Proxy({ exec: function() { return null; } }, { get: function(o, k) { get.push(k); return o[k]; }});
         RegExp.prototype[Symbol.search].call(p);
-        return get + '' === "lastIndex,exec";
+        return get + '' === "lastIndex,exec,lastIndex";
       */},
+      note_id: "regexp-prototype-symbolsearch",
```

Comment:
i believe the notes have to go on a "tr" key inside "res" (or on individual results), as opposed to just top-level here.